css/donate.css

```diff
@@ -75,57 +81,53 @@ h2 {
   object-fit: contain; }
 
 .donate-impact {
-  background-color: #ece3f2;
+  background-color: rgba(236, 177, 206, 0.26);
   padding-top: 3%;
   padding-bottom: 3%; }
 
 .impact-header {
+  color: black;
   text-align: center;
-  padding-bottom: 1%; }
+  padding-bottom: 2.5%; }
 
-.donate-option1 {
-  background: #ffffff;
-  border: 6px solid #1cd5d5;
-  padding: 3%;
-  padding-top: 5%;
-  text-align: center;
-  width: 100%;
-  height: 100%; }
+.donate-option1-header, .donate-option2-header, .donate-option3-header {
+  display: inline-block;
+  font-size: 300%;
+  margin-bottom: 10%; }
 
 .donate-option1-header {
-  color: #1cd5d5;
-  font-size: 300%; }
-
-.donate-option2 {
-  background: #ffffff;
-  border: 6px solid #a88aa8;
-  padding: 3%;
-  padding-top: 5%;
-  text-align: center;
-  width: 100%;
-  height: 100%; }
+  color: #82ecec; }
```

Comment:
can you update this blue to match that of the first two social buttons & intercom bot? #6fcbdc